mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -44,23 +53,28 @@
     @NotNull
     public static MesosSchedulerClient<Call, Protos.Event> usingProtos(
         @NotNull final String host,
-        final int port
+        final int port,
+        @NotNull final String name
     ) {
-        return new MesosSchedulerClient<>(host, port, "rx-mesos", MessageCodecs.PROTOS_CALL, MessageCodecs.PROTOS_EVENT);
+        return new MesosSchedulerClient<>(host, port, name, MessageCodecs.PROTOS_CALL, MessageCodecs.PROTOS_EVENT);
     }
 
 
     @NotNull
-    private final MessageCodec<Send> sendCodec;
+    @VisibleForTesting
+    final MessageCodec<Send> sendCodec;
 
     @NotNull
-    private final MessageCodec<Receive> receiveCodec;
+    @VisibleForTesting
+    final MessageCodec<Receive> receiveCodec;
 
     @NotNull
+    @VisibleForTesting
     private final HttpClient<ByteBuf, ByteBuf> httpClient;
 
     @NotNull
-    private final Func1<Send, Observable<HttpClientRequest<ByteBuf>>> createPost;
+    @VisibleForTesting
+    final Func1<Send, Observable<HttpClientRequest<ByteBuf>>> createPost;
```

Comment:
It doesn't look like the above fields are referenced in the tests you added in this PR. Was there another reason for making them non-private and marked `@VisibleForTesting`?